webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -340,7 +365,14 @@ const TableActionBar: React.FC<Props> = ({
         </Space>
       </Column>
       <Column align="right">
-        {!!toggleComparisonView && <Button onClick={toggleComparisonView}>Compare</Button>}
+        <Space>
+          <Dropdown content={settingContent}>
+            <Button>
+              <Icon name="overflow-horizontal" title="menu" />
```

Comment:
nit: An overflow menu with no label is pretty vague on what it includes. I would also expect an overflow menu to be on the right-hand side of the column instead of the left. I also prefer putting the icon in the `icon` prop of the button, but that's just a personal preference.

Reply:
We are going to change the overflow icon to something else when available, but I think it will be at this same location according to design